### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by hillaryj — APPROVED** [review]
⚡

**Review by jsf9k — APPROVED** [review]
:zap:

**Review by felddy — DISMISSED** [review]
🦾

**Review by felddy — CHANGES_REQUESTED** [review]
We missed a `set-env`.

**Review by felddy — APPROVED** [review]
Looks good now.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -53,8 +54,9 @@ jobs:
           python-version: 3.8
       - name: Store installed Python version
```
Comment (dav3r) [inline .github/workflows/build.yml:55]: Don't we also want to update the `python-version` here to `3.9` as well?

**Review by dav3r — CHANGES_REQUESTED** [review]
Please see my comment before merging - thanks!

File: .github/workflows/build.yml (hunk 1 above)
Comment (jsf9k) [reply .github/workflows/build.yml:55]: Yes.  See commit 2ca9b0a.

**Review by dav3r — APPROVED** [review]
👍 👍

**Pull request merged**